applications/DelaunayMeshingApplication/custom_processes/settle_model_structure_process.hpp

```diff
@@ -923,12 +923,12 @@ class SettleModelStructureProcess
 
 
     //Sort
-    // rComputingModelPart.Nodes().Sort();
+    rComputingModelPart.Nodes().Sort();
```

Comment:
`Unique()` does a `Sort()` inside, maybe this can be commented?

Reply:
Ok thank you!